ui/src/pages/domain/[domain]/service/[service]/tags.js

```diff
@@ -0,0 +1,222 @@
+                        <title>Athenz Service Tags</title>
+                    </Head>
+                    <Header showSearch={true} />
+                    <MainContentDiv>
+                        <AppContainerDiv>
+                            <TagsContainerDiv>
+                                <TagsContentDiv>
+                                    <PageHeaderDiv>
+                                        <NameHeader
+                                            category={'service'}
+                                            domain={domainName}
+                                            collection={serviceName}
+                                            collectionDetails={
+                                                serviceDetails
+                                                    ? serviceDetails
+                                                    : {}
+                                            }
+                                        />
+                                        <CollectionDetails
+                                            collectionDetails={
+                                                serviceDetails
+                                                    ? serviceDetails
+                                                    : {}
+                                            }
+                                            _csrf={_csrf}
+                                        />
+                                        <ServiceTabs
+                                            domain={domainName}
+                                            service={serviceName}
+                                            selectedName={'tags'}
+                                        />
+                                    </PageHeaderDiv>
+                                    <TagList
+                                        collectionDetails={serviceDetails}
+                                        domain={domainName}
+                                        collectionName={serviceName}
+                                        tags={serviceTags}
+                                        category={'service'}
+                                        _csrf={this.props._csrf}
+                                    />
+                                </TagsContentDiv>
+                            </TagsContainerDiv>
+                            <UserDomains domain={domainName} />
+                        </AppContainerDiv>
+                    </MainContentDiv>
+                </div>
+            </CacheProvider>
+        );
+    }
+}
+
+const mapStateToProps = (state, props) => {
+    return {
+        ...props,
+        isLoading: selectIsLoading(state),
+        serviceDetails: selectService(
+            state,
+            props.domainName,
+            props.serviceName
+        ),
+        serviceTags: selectServiceTags(
+            state,
+            props.domainName,
+            props.serviceName
+        ),
+    };
```

Comment:
can we get a screen recording of the new page linked to this pr? Would like to have a visual for better review.

Reply:
added 2 screenshots to the description.